src/server/ApiDispatchers.cpp

```diff
@@ -781,14 +755,11 @@ static DWORD TraceGetThreadId(CONSOLE_API_MSG* const m)
     switch (a->StringType)
     {
     case CONSOLE_ATTRIBUTE:
-        Telemetry::Instance().LogApiCall(Telemetry::ApiCall::ReadConsoleOutputAttribute);
         break;
     case CONSOLE_ASCII:
-        Telemetry::Instance().LogApiCall(Telemetry::ApiCall::ReadConsoleOutputCharacter, false);
         break;
     case CONSOLE_REAL_UNICODE:
     case CONSOLE_FALSE_UNICODE:
-        Telemetry::Instance().LogApiCall(Telemetry::ApiCall::ReadConsoleOutputCharacter, true);
         break;
     }
 

```

Comment:
wait somebody's gonna find this in the future and choke on their Soylent

```c++
switch (a->StringType)
{
	case 1:
		break;
	case 2:
		break;
	case 3:
		break;
}
```

complicated no-op. it keeps our enemies off our trail